theseus/geometry/manifold.py

```diff
@@ -40,7 +41,14 @@ def __init__(
         if tensor is None and dtype is None:
             dtype = torch.get_default_dtype()
         if tensor is not None:
-            tensor = self._check_tensor(tensor, strict)
+            if _LieGroupCheckContext.get_context():
+                tensor = self._check_tensor(tensor, strict)
+            else:
+                warnings.warn(
+                    f"functorch is enabled and tensor is not checked "
```

Comment:
We should change this message as well, since this now has nothing to do with `functorch`. Maybe "Lie group consistency checks are disabled."? Feel free to change in #268 if that's easier.

Reply:
Oh yeah, good catch!